project/webapp/test/integration/pages/ProductsList.js

```diff
@@ -1,28 +1,100 @@
-sap.ui.define([
-	"sap/ui/test/Opa5"
-], function (Opa5) {
-	"use strict";
-	var sViewName = "ProductsList";
-	
-	Opa5.createPageObjects({
-		onTheViewPage: {
+sap.ui.define(
+  [
+    "sap/ui/test/Opa5",
+    "sap/ui/test/actions/Press",
+    "sap/ui/test/matchers/Properties",
+    "sap/ui/test/matchers/AggregationLengthEquals",
+    "sap/ui/test/matchers/BindingPath"
+  ],
+  function (Opa5, Press, Properties, AggregationLengthEquals, BindingPath) {
+    "use strict";
+    var sViewName = "ProductsList";
 
-			actions: {},
+    Opa5.createPageObjects({
+      onTheProductsList: {
+        actions: {
+          iSelectListItem() {
+            return this.waitFor({
+              controlType: "sap.m.CheckBox",
+              viewName: sViewName,
+              actions: new Press(),
+              matchers: new BindingPath({
+                path: "/products/0",
+                modelName: "appModel"
+              }),
+              success() {
+                Opa5.assert.ok(true, "The first product is selected");
+              },
+              errorMessage: "The table does not have a trigger"
+            });
+          },
+          iPressOnDeleteButton() {
+            return this.waitFor({
+              id: "deleteProductsBtn",
+              viewName: sViewName,
+              actions: new Press(),
+              success() {
+                Opa5.assert.ok(true, "The Delete button is pressed");
+              },
+              errorMessage: "Can't press delete button"
+            });
+          },
+          iClickOnConfirmDeleteButton() {
+            return this.waitFor({
+              viewName: sViewName,
+              searchOpenDialogs: true,
+              controlType: "sap.m.Button",
+              matchers: new Properties({
+                text: "OK"
```

Comment:
do not hard code action names as strings
import sap.m.MessageBox and use here MessageBox.Action.OK